scss/components/_flex.scss

```diff
@@ -32,3 +32,30 @@
     }
   }
 }
+
+@mixin gimbal($o:row,$x:left,$y:top) {
+  //columns
+  $map-xy: (
+  right: bottom,
+  left: top,
+  top: left,
+  bottom: right,
+  center: middle,
+  middle: center
+);
```

Comment:
Oh, I know. I will fix this at the end -- after all the technical stuff is sorted out.